packages/core/saved-objects/core-saved-objects-server-internal/src/routes/legacy_import_export/export.ts

```diff
@@ -31,6 +31,7 @@ export const registerLegacyExportRoute = (
         }),
       },
       options: {
+        access: 'public',
```

Comment:
Technically this is public, because it is documented as such BUT since this is scheduled to be deleted for 9.0 we could also leave this unchanged for the remainder of 8.0